src/solver/surfaceIntegrations.F90

```diff
@@ -568,22 +575,69 @@ subroutine wallIntegrationFace(localValues, mm)
             v(3) = ww2(i, j, ivz)
             v = v / (sqrt(v(1)**2 + v(2)**2 + v(3)**2) + 1e-16)
 
-            ! Dot product with free stream
-            sensor = -(v(1) * velDirFreeStream(1) + v(2) * velDirFreeStream(2) + &
-                       v(3) * velDirFreeStream(3))
+            if (sepmodel == surfvec) then
+                ! Freestream projection over the surface.
+                vectTangentialNorm = velDirFreeStream(1) * BCData(mm)%norm(i, j, 1) + &
```

Comment:
Isn't this the freestream velocity projected in the surface normal direction? I don't understand why it is called `vectTangentialNorm`.

Reply:
Right, its the dot prdocut of freestream and the normal vector. I changed the name